cmd/podman/machine/init.go

```diff
@@ -106,8 +107,18 @@ func init() {
 	flags.StringVar(&initOpts.IgnitionPath, IgnitionPathFlagName, "", "Path to ignition file")
 	_ = initCmd.RegisterFlagCompletionFunc(IgnitionPathFlagName, completion.AutocompleteDefault)
 
-	rootfulFlagName := "rootful"
-	flags.BoolVar(&initOpts.Rootful, rootfulFlagName, false, "Whether this machine should prefer rootful container execution")
+	rootfullFlagName := "rootfull"
+	flags.BoolVar(&initOpts.Rootfull, rootfullFlagName, false, "Whether this machine should prefer rootfull container execution")
+	flags.SetNormalizeFunc(aliasFlags)
```

Comment:
Should we still support `rootful`?  Similar to how we're still supporting the `--pull-always` option under the covers?

Reply:
We do, that is what the SetNormalizeFunc does,  It basically aliases rootful->rootfull, so if people built scripts with the old option, it will continue to work.